app/experimenter/nimbus-ui/src/components/FormOverview/index.test.tsx

```diff
@@ -206,26 +210,27 @@ describe("FormOverview", () => {
       "*": "Big bad happened",
     };
     render(<Subject {...{ submitErrors }} />);
-    await act(async () => {
+    await act(async () =>
       expect(screen.getByTestId("submit-error")).toHaveTextContent(
         submitErrors["*"],
-      );
-    });
+      ),
+    );
   });
 
   it("displays feedback for per-field error", async () => {
     const submitErrors = {
       name: "That name is terrble, man",
     };
     render(<Subject {...{ submitErrors }} />);
+    const errorFeedback = screen.getByText(submitErrors["name"]);
     await act(async () => {
-      const errorFeedback = screen.getByText(submitErrors["name"]);
       expect(errorFeedback).toHaveClass("invalid-feedback");
       expect(errorFeedback).toHaveAttribute("data-for", "name");
     });
   });
 });
 
+// TODO: EXP-462 Get this from constants / config loaded at app start?
 const APPLICATIONS = ["firefox-desktop", "fenix", "reference-browser"];
```

Comment:
Not going to get the runtime app config in a test and constants won't do - this should be mocked here

Reply:
Good point, with #3851 now merged the mocked cache now has a mock config loaded that should be useful.